src/utils/Wallet.ts

```diff
@@ -315,10 +315,10 @@ export class Wallet extends Program {
             })
         }
 
+        // change no need memo
```

Comment:
no need => memo is unnecessary for fee

Reply:
done 
[45a030a](https://github.com/hashgreen/hoogii-wallet/pull/19/commits/45a030a02e34176c57a35cae449ddc3bd94a74d0)